fe/fe-core/src/main/java/org/apache/doris/master/ReportHandler.java

```diff
@@ -296,32 +288,27 @@ private static void tabletReport(long backendId, Map<Long, TTablet> backendTable
             deleteFromMeta(tabletDeleteFromMeta, backendId, backendReportVersion);
         }
 
-        // 4. handle (be - meta)
-        if (foundTabletsWithValidSchema.size() != backendTablets.size()) {
-            deleteFromBackend(backendTablets, foundTabletsWithValidSchema, foundTabletsWithInvalidSchema, backendId);
```

Comment:
Why delete this code? If it is removed, how to deal with the rubbish tablet on BE?

Reply:
my fault， I will fix it